edb/pgsql/metaschema.py

```diff
@@ -1478,6 +1478,41 @@ def __init__(self) -> None:
             text=self.text)
 
 
+class DurationInFunction(dbops.Function):
+    """Cast text into duration, ensuring there is no month units"""
+    text = r'''
+        SELECT
+            CASE WHEN
+                EXTRACT(MONTH FROM v.column1) != 0 OR
+                EXTRACT(YEAR FROM v.column1) != 0 OR
+                EXTRACT(DAY FROM v.column1) != 0
+            THEN
+                edgedb._raise_specific_exception(
+                    'invalid_datetime_format',
+                    'invalid input syntax for type duration: '
+                        || quote_literal(val),
+                    '{"hint":"You can''t use days or units ' ||
```

Comment:
Please use neutral style here: "Units bigger than days cannot be used for std::duration.".  Also, [PostgreSQL error style guide](https://www.postgresql.org/docs/current/error-style-guide.html) is excellent.